pkgs/tools/security/fail2ban/0_10.nix

```diff
@@ -0,0 +1,7 @@
+{ callPackage, ... } @ args:
+
+callPackage ./. (args // {
+  version = "0.10.4";
+  rev     = "version";
```

Comment:
this works on your machine because you already have something with this sha256 fetched, but "version" is not the right rev